pages/learn/develop/blocks.md

```diff
@@ -111,14 +111,61 @@ By default, the block is set to `track latest`, which means that new releases ar
 
 ## Making your block public
 
-When a block is created on balenaCloud it will not automatically be publicly visible on balenaHub. When a block is made public it is listed in the [Blocks section][blockhub] of balenaHub, but is not joinable like fleets or forkable like projects. Instead the card for the block allows users to find the source code for the block, and learn how to deploy and use it in their own fleet.
+When a block is created on balenaCloud it will not automatically be publicly visible on balenaHub. When a block is made public it is listed in the [Blocks section][blockhub] of balenaHub, but is not joinable like fleets or forkable like projects. Instead the card for the block allows users to find the image URL for the block, and learn how to deploy and use it in their own fleet.
```

Comment:
```suggestion
When a block is created on balenaCloud, it won't automatically be visible on balenaHub. When the block's visibility is toggled to `public`, it will get listed in the [Blocks section][blockhub] of balenaHub. Blocks aren't joinable like fleets or forkable like projects. Instead, the balenaHub page would provide the container image URL for the block, and instructions on how to deploy and use the block in your own fleet.
```